docs/high-availability/srv_kubernetes/apiserver.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
are these being included twice or are they different? if so, what is the difference?